### PR TITLE
Prepopulate bash history for developer convenience

### PR DESCRIPTION
**Pull request by findepi**

### CODE REVIEW

File: prestodev/hdp2.6-hive/Dockerfile [hunk 1]
```diff
@@ -58,6 +58,12 @@ RUN ssh-keygen -t rsa -b 4096 -C "automation@prestosql.io" -N "" -f /root/.ssh/i
 RUN chmod 755 /root && chmod 700 /root/.ssh
 RUN passwd --unlock root
 
+# Provide convenience bash history
+RUN set -xeu \
```
Comment (kokosing) [inline prestodev/hdp2.6-hive/Dockerfile]: can you please do same for kerberized environments?

File: prestodev/hdp3.1-hive/files/etc/hive/conf/beeline-site.xml [hunk 2]
```diff
@@ -1,17 +0,0 @@
...
-    <property>
-        <name>beeline.hs2.jdbc.url.httpUrl</name>
-        <value>jdbc:hive2://localhost:10000/default;user=hdfs;password=hive;transportMode=http;httpPath=cliservice</value>
-    </property>
-
-    <property>
-        <name>beeline.hs2.jdbc.url.default</name>
-        <value>tcpUrl</value>
-    </property>
-</configuration>
```
Comment (findepi) [inline prestodev/hdp3.1-hive/files/etc/hive/conf/beeline-site.xml]: @stagraqubole is this config file needed?

Comment (stagraqubole) [reply prestodev/hdp3.1-hive/files/etc/hive/conf/beeline-site.xml]: This is needed for the tests to run. Without this, tests will not run as beeline (which is by default used when hive cli is invoked in hive3) will not have right parameters to connect to server.

File: prestodev/hdp2.6-hive/Dockerfile (hunk 1 above)
Comment (findepi) [reply prestodev/hdp2.6-hive/Dockerfile]: do you have particular commands in mind? kinit & stuff?

Comment (findepi) [reply prestodev/hdp2.6-hive/Dockerfile]: added some

**Comment by findepi** [comment]
Superseded by https://github.com/prestosql/docker-images/pull/23

**Pull request closed without merging**